package com.namelessCodeF.codeforces.exception;

public class ResultProcessingException extends RuntimeException {
    public ResultProcessingException(String message, Throwable cause) {
        super(message, cause);
    }
}
